cpp/src/arrow/array/builder_binary.h

```diff
@@ -295,6 +295,11 @@ class ARROW_EXPORT FixedSizeBinaryBuilder : public ArrayBuilder {
   int32_t byte_width_;
   BufferBuilder byte_builder_;
 
+  /// Temporary access to a value.
+  ///
+  /// This pointer becomes invalid on the next modifying operation.
+  uint8_t* GetMutableValue(int64_t i);
```

Comment:
Wouldn't it be better to make this inline? Otherwise `Decimal128Builder::UnsafeAppend` will have to do a function call for this trivial operation.

Reply:
Yes, done.